### DISCUSSION TITLE
limactl start process?

### DISCUSSION THREAD

**Discussion by ttc0419** [lima-vm/lima#2721] [Q&A]
I'm trying to make my own custom alpine images. I removed the cloud-init, and save the interfaces in /etc/network. SSH seems fine, but it stuck at step two:
```
INFO[0009] [hostagent] Waiting for the essential requirement 1 of 2: "ssh" 
INFO[0010] [hostagent] The essential requirement 1 of 2 is satisfied 
INFO[0010] [hostagent] Waiting for the essential requirement 2 of 2: "user session is ready for ssh" 
INFO[0049] [hostagent] Waiting for the essential requirement 2 of 2: "user session is ready for ssh" 
INFO[0089] [hostagent] Waiting for the essential requirement 2 of 2: "user session is ready for ssh"
```
What's the setup like when we run the limactl start command, how to make an image that does not depend on cloud init?

**Comment by jandubois** [comment]
Lima always depends on "something" that processes the `user-data` file generated for `cloud-init`.

I've written `lima-init` as a `cloud-init` replacement that works on a minimal Alpine image; see the https://github.com/lima-vm/alpine-lima repo for how it is used in our Alpine ISO.
